magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/services/EC2Discovery.java

```diff
@@ -365,23 +365,28 @@ useful to know this, but we dont need to alert on it, so send a segment (vs sent
       logger.error("Error while getting IP owner for {}", cidr, e);
       //TODO: really want to move this sentry event into shared discovery exceptions
       //but the refactor of that class is in an active branch - will pick it up later
-      Sentry.capture(new EventBuilder().withMessage("WHOIS owner lookup exception while running EC2 SecurityGroup discovery")
-        .withLevel(Event.Level.ERROR)
-        .withExtra("Resource", resourceARN)
-        .withExtra("CIDR", cidr)
-        .withExtra("Reason", e)
-        .withSentryInterface(new ExceptionInterface(e)));
+        SentryEvent event = new SentryEvent(e);
```

Comment:
Perhaps a util method to send the Sentry Events since we're doing it 3 times and possibly more in the future?  Something like sendSentryEvent(String message, SentryLevel level, Map<String, Object> extras)?  Similar to what's in DiscoverExecptions.java.

Reply:
My goal was to migrate the existing code which built the events in place. Are you asking me to extract a common method in addition to the update to the dependency?

Reply:
My line of thinking is, if not now, when should we do it?  But if that work requires a ticket, then yes it would be out of scope.